### ISSUE TITLE
Publish to OPAM

### ISSUE THREAD

**Issue by vkurchatkin** [uber/NEAL#6]
Are you planning on publishing NEAL to OPAM?

**Comment by tadeuzagallo** [comment]
Yes, thanks for the reminder. I've opened a PR on the opam repository repo: https://github.com/ocaml/opam-repository/pull/10400
